src/auth.service.ts

```diff
@@ -143,7 +143,7 @@ export class AuthService {
     private _fetchGoogleUserDetails(){
         let currentUser = this.gauth.currentUser.get();
         let profile = currentUser.getBasicProfile();
-        let idToken = currentUser.getAuthResponse().id_token;
+        let idToken = currentUser.getAuthResponse().access_token;
```

Comment:
You can't just change value of return, if you want to add something, create new variable named accessToken, assign value to it and place in return function, so you'll have possibility to operate with idToken and accessToken after success login